website/src/components/announcement.js

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable no-undef */
+import React, { Component } from "react";
+import styled from 'styled-components';
+import CloseButton from "./close-button";
+
+const AnnounceBanner = styled.div`
+  display: flex;
+  position: fixed;
+  width: 100vw;
+  height: 64px;
+  bottom: 0;
+  z-index: 3;
+  background: ${props => props.theme.colors.primary};
+  padding: 14px 24px;
+  justify-content: space-between;
+
+  @media screen and (max-width: ${props => props.theme.breakpoints.medium}px) {
+    height: 200px;
+    padding-top: 44px;
+
+    button {
+      position: absolute;
+      top: 8px;
+      right: 8px;
+    }
+  }
+`
+
+const AnnounceContent = styled.div`
+  display: flex;
+  align-items: center;
```

Comment:
it should be center aligned:

<img width="1751" alt="image" src="https://user-images.githubusercontent.com/1161870/189972403-211da0aa-9d2c-4d1a-8e90-03b5023c7113.png">